internal/routing/handlers/user_handler.go

```diff
@@ -437,3 +443,308 @@ func generateToken() string {
 	// Generate a random 6-digit number
 	return strconv.Itoa(rand.Intn(900000) + 100000)
 }
+
+func (handler *UserHandler) HandleGetUserRequest(w http.ResponseWriter, r *http.Request) {
+	ctx, cancel := context.WithDeadline(r.Context(), time.Now().Add(500*time.Second))
```

Comment:
500 Sekunden? that's a lot, da braucht man auch keine deadline mehr, lmao

Reply:
Hat sich mein Debug eingeschlichen 😙